src/Crudit/CrudMenu/HermesMenuProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Lle\HermesBundle\Crudit\CrudMenu;
+
+use Lle\CruditBundle\Contracts\MenuProviderInterface;
+use Lle\CruditBundle\Dto\Icon;
+use Lle\CruditBundle\Dto\Layout\LinkElement;
+use Lle\CruditBundle\Dto\Path;
+
+/**
+ * Class HermesMenuProvider
+ * @package Lle\HermesBundle\Crudit\CrudMenu
+ *
+ * @author 2LE <2le@2le.net>
+ */
+class HermesMenuProvider implements MenuProviderInterface
+{
+    public function getMenuEntry(): iterable
+    {
+        /** @var LinkElement $menu */
+        $menu = LinkElement::new(
+            'menu.lle_hermes.hermes',
+            null,
+            Icon::new('/img/icons/groupes.svg', Icon::TYPE_IMG),
+        )->setRole('ROLE_LLE_HERMES');
+        $menu->addChild(
```

Comment:
on doit mettre setParent et pas addChild. Sinon, impossible d'ajouter des entrée dans parametre par exemple.

Reply:
That impossible because setParent ask a string that mean Crudit have a bug